src/include/OpenImageIO/ustring.h

```diff
@@ -815,23 +815,32 @@ class OIIO_UTIL_API ustringhash {
     }
 
     /// Construct a ustringhash from a null-terminated C string (char *).
-    OIIO_HOSTDEVICE explicit ustringhash(const char* str)
+    OIIO_DEVICE_CONSTEXPR explicit ustringhash(const char* str)
```

Comment:
Maybe add a comment on why the CPU side can't be constexpr.

Reply:
done